src/main/java/org/owasp/benchmark/score/BenchmarkScore.java

```diff
@@ -48,37 +48,7 @@
 import javax.xml.parsers.DocumentBuilderFactory;
 
 import org.apache.commons.io.FileUtils;
-import org.owasp.benchmark.score.parsers.AcunetixReader;
```

Comment:
wow, none of the changes made to this file are relevant to adding your new reader

Reply:
@dbolkensteyn  i saw unused imports, wrong tabs and as a good boy scout i fixed the format of the source and cleaned up it. If you want to i can revert those changes. Thank you!